server/common/common/urls.py

```diff
@@ -15,8 +15,18 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from django.conf.urls import url
```

Comment:
Alphabetically sort all import:

from django.conf import settings
from django.conf.urls import url
from django.contrib import admin
from django.urls import include, path
from django.views.static import serve

Reply:
I sorted all import in alphabetical order